app/wrappers/wakes/facebook_metrics_wrapper.rb

```diff
@@ -22,7 +22,7 @@ def share_counts
           hash[r['id']] = r.fetch('share').fetch('share_count')
         rescue KeyError
           # Some urls can have no 'share' keys. This is generally in the case when the url isn't valid.
-          Rails.logger.warn "Share count not received for #{r['id']}. Possibly an invalid url?"
+          Rails.logger.warn "Share count not received for #{r['id']}. Response is: #{r}"
```

Comment:
@gaganawhad what kind of object is `r`? There are response objects out there that could be thousands and thousands of characters. I don't think getting all that in our logs is helpful or useful. Do you know what the average response object is?